core/Files.h

```diff
@@ -126,11 +131,14 @@ class File final {
     StrictLevel strictLevel;
 
     const CompiledLevel compiledLevel;
+    const PackagedLevel packagedLevel;
 
 private:
     std::shared_ptr<const FileHash> hash_;
 };
-CheckSize(File, 96, 8);
+
+// TODO: see if we can pack `packagedLevel` in a bit better to get this back down to 96
+CheckSize(File, 104, 8);
```

Comment:
A fresh challenge for you, @froydnj :D

Reply:
There should be two spare bytes right after `flags` which will accommodate this. :D